apps/server/minecraft_manager/players/service.py

```
@@ -41,6 +41,18 @@ def observe_presence(self, player: str, connected: bool, xuid: str = "") -> None
+        if connected:
+            mode = self.repository.get_preferred_game_mode(player)
+            if mode is not None:
+                try:
+                    self.console.set_game_mode(player, mode)
+                    self.events.publish(
+                        "player.game_mode.reapplied",
+                        "bedrock-log",
+                        {"player": player, "mode": mode, "origin": "auto-reapply"},
+                    )
+                except Exception:
+                    pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)